spring-content-rest/src/main/java/org/springframework/data/rest/extensions/entitycontent/RepositoryEntityMultipartController.java

```diff
@@ -97,7 +97,8 @@ public ResponseEntity<RepresentationModel<?>> createEntityAndContent(RootResourc
 
         String store = pathSegments[1];
 
-        boolean entitySaved = false;
+        // Save the entity and re-assign the result to savedEntity, so that its @Id property exists
```

Comment:
```suggestion
        // Save the entity and re-assign the result to savedEntity, so that it exists in the repository before content is added to it.
```

Just a suggestion to make it more clear what we want to be doing here: making sure that it exists before content is added